src/js/tabs/send.js

```diff
@@ -219,18 +223,25 @@ SendTab.prototype.angular = function (module)
               $scope.sendForm.send_destination.$setValidity("federation", false);
```

Comment:
While you're at cleaning up the code style, why not use single quotes here, too? :wink: